### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by skyfaller** [mcobzarenco/zee#2]
Thanks for writing an Emacs-ish editor in Rust! Due to an accident of computer science education, I prefer Emacs keybindings over vim bindings.

Could you [add a license for your code to your repo](https://help.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository), to make it clear that people are welcome to contribute?

I personally prefer the AGPL. I agree with [Bruce Perens saying there are too many licenses](https://www.datamation.com/osrc/article.php/12068_3803101_2/Bruce-Perens-How-Many-Open-Source-Licenses-Do-You-Need.htm) creating license compatibility problems. That said, I'm ok with any license approved by the FSF + OSI. (Not that my opinion matters, since I'm just a random dude on the internet.)

**Comment by mcobzarenco** [comment]
@skyfaller Thanks for the kind words.

Good point re: adding a licence -- I'd typically prefer something like MIT -- AGPL rings bells re: whether commercial use is ok without sharing the code, but I'm not sure if that's correct :shrug:

**Comment by skyfaller** [comment]
Were you planning on selling copies of zee without distributing the source code? I can see that the AGPLv3 might interfere with that, but it seems like a strange proposal in this day and age. Personally I would be much more worried that some company like Apple would start using your software in their computers (or Amazon on their servers) without contributing their improvements back to you.

If you do prefer a permissive license, I think the Apache License 2 is better than MIT/BSD because it protects you a bit better from software patent lawsuits.

**Comment by mcobzarenco** [comment]
> Were you planning on selling copies of zee without distributing the source code?

Never, god forbid! :-) 

I was thinking more of the case you described -- not hindering a company using a modified version without sharing the code. 

> If you do prefer a permissive license, I think the Apache License 2 is better than MIT/BSD because it protects you a bit better from software patent lawsuits.

:+1:  Actually, looking closer, I noticed I've already have a `license = "Apache-2.0"` field in `Cargo.toml`. 

I just merged a commit to:
 1. change the license to dual MIT or Apache 2.0 [similar to rust-lang](https://internals.rust-lang.org/t/rationale-of-apache-dual-licensing/8952)
 2. Added the two licenses in the repo root: `LICENSE-APACHE` and `LICENSE-MIT`
